packages/react/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
   "extends": "./tsconfig.build.json",
-  "include": ["src/**/*.ts", "test/**/*.ts"],
+  "include": ["src/**/*.ts?(x)", "test/**/*.ts?(x)"],
```

Comment:
Was testing this, and this pattenr matching wasn't working well for me (had ts error), but this worked:

```
"include": ["src/**/*.ts","src/**/*.tsx", "test/**/*.ts", "test/**/*.tsx"],
  "compilerOptions": {
    "jsx": "preserve"
  },
```

Reply:
Hmm, interesting, changed it in 88ec9072e3b7b74d6b84aeecc3e866d7296db239 to make sure it works for everyone, thanks!